index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title>Mo Uddin Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <div class="navbar">
```

Comment:
The only thing in the head should be metadata - no elements that you actually want to be on the page.